File: frontend/src/interfaces/Dormitory.ts
export interface DormitoryInterface {

    ID?: number;
    DormName?: string;    
    DormDescription?: string;           
    DormEquipment?: string;    
    DormPic: Uint8Array; // ใช้ `Uint8Array` สำหรับจัดการไฟล์ใน JavaScript
    Price?: number;
  }

File: backend/entity/Dormitory.go
package entity

import (
	"gorm.io/gorm"
)

type Dormitory struct {
	gorm.Model
	DormName  string     
	DormDescription    string            
	DormEquipment      string    
	DormPic      []byte           
	Price      int   
	RoomID uint   	 
}